packages/storybook/src/schematics/ng-add/ng-add.ts

```diff
@@ -0,0 +1,27 @@
+import { chain, Rule } from '@angular-devkit/schematics';
+import { addDepsToPackageJson } from '@nrwl/workspace';
+import {
+  babelCoreVersion,
+  babelLoaderVersion,
+  storybookAddonKnobsVersion,
+  storybookAngularVersion
+} from '../../utils/versions';
+import { Schema } from './schema';
+
+export default function(schema: Schema) {
+  return chain([updateDependencies()]);
+}
+
+function updateDependencies(): Rule {
+  return addDepsToPackageJson(
+    {},
+    {
+      '@storybook/angular': storybookAngularVersion,
+      '@storybook/addon-knobs': storybookAddonKnobsVersion,
```

Comment:
This seems like some plugin... Should we let users add this for themselves if they want to?

Reply:
I'm using this plugin for the core functionality of creating isolated cypress ui tests.  Eventually, I'd like to give people the choice to include other plugins automatically.